digitalpublicgoods/opensrp.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "OpenSRP",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "We recently trademarked OpenSRP through support from Digital Square.  We have not published this online but can provide this information via email if needed."
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://smartregister.atlassian.net/wiki/spaces/Documentation/overview"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "We have Beam connectors that can do this."
```

Comment:
Please include link to supporting documentation.

Reply:
Resolved in commit 3e119f4